{"version":"3.6.0","examples":[{"id":"./spec/pry_debugging_spec.rb[1:1]","description":"takes in a number as an argument and returns the sum of that number and 2","full_description":"#plus_two takes in a number as an argument and returns the sum of that number and 2","status":"passed","file_path":"./spec/pry_debugging_spec.rb","line_number":5,"run_time":264.439731213,"pending_message":null}],"summary":{"duration":264.443196068,"example_count":1,"failure_count":0,"pending_count":0},"summary_line":"1 example, 0 failures"}